src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/FinancialTransactionCategoryCreateDTO.java

```diff
@@ -7,4 +7,4 @@
 import javax.validation.constraints.Size;
 
 public record FinancialTransactionCategoryCreateDTO(@NotBlank @Size(max = 30, message = "CATEGORY NAME TOO LONG") String name,
-                                                    @NotNull FinancialTransactionType type){}
+                                                    @NotNull FinancialTransactionType type, @NotNull Long userId){}
```

Comment:
Code formatting.
Before committing code strongly recommended to use the Checkstyle plugin. The config for our project can be found in:
https://github.com/bycza-zagroda/track-expenses-app-backend/blob/develop/checkstyle/tea.xml

Reply:
Done.